src/main/java/duke/logic/commands/results/Imageable.java

```diff
@@ -0,0 +1,8 @@
+package duke.logic.commands.results;
+
+import javafx.scene.image.Image;
+
+public interface Imageable {
```

Comment:
What is imageable?

Reply:
We are unsure of how to name the interface, it is to meant the class that implements the interface should contains an Image and a method to get the image